### ISSUE TITLE
Setup library scanning should be nonblocking

### ISSUE THREAD

**Issue by kowdermeister** [beresmate/nodeselekta#5]
The scanning for ID3 tags and dumping them to DB should be a non-blocking process. The scanner will run as a background process, while the app is already functional.

Pros:

 * nice UX
 * app can start instantly, user is not lost in the flow

Cons: 

 * user faces folders and filenames
 * tracks are *streamed in* as they are scanned (could be avoided if UUID is already generated)

**Comment by beresmate** [comment]
Yup, will refactor the API once we're done with the first alpha UI design :)
